app/controllers/insured/plan_shoppings_controller.rb

```diff
@@ -443,7 +443,13 @@ def is_eligibility_determined_and_not_csr_0?(entity)
   def send_receipt_emails
     email = @person.work_email_or_best
     UserMailer.generic_consumer_welcome(@person.first_name, @person.hbx_id, email).deliver_now
-    return unless EnrollRegistry.feature_enabled?(:send_secure_purchase_confirmation_email)
+    send_email_enabled = if @enrollment.is_shop?
```

Comment:
can we rename the variable `send_email_enabled` to something like `send_email_if_feature_enabled` maybe?

Reply:
hmm - i thought to keep it short. but, may be `is_confirmation_email_enabled` more meaningful i guess.